source/_integrations/cloudflare.markdown

```diff
@@ -4,6 +4,7 @@ description: Automatically update your Cloudflare DNS records.
 ha_category:
   - Network
 ha_release: 0.74
+ha_iot_class: ~
```

Comment:
This integration pushes data to the cloud, right?

```suggestion
ha_iot_class: Cloud Push
```

Reply:
Oh darn, I thought that only applied to retrieving the state. My bad, thanks :+1: